src/civic/index.js

```diff
@@ -590,34 +602,38 @@ const downloadEvidenceRecords = async (baseUrl, trustedCurators) => {
 const upload = async ({
     conn, errorLogPrefix, trustedCurators, ignoreCache = false, maxRecords, url = BASE_URL,
 }) => {
-    // add the source node
+    // 1. ADDING CIVIC SOURCE TO GRAPHKB
     const source = await conn.addSource(SOURCE_DEFN);
 
+    // 2. FETCHING PREVIOUS RECORDS FROM GRAPHKB
```

Comment:
are the numbered comments leftover from debugging? They seem superfluous with the other inline comments

Reply:
It is how I learned to code but I took them out for uniformity, np.